### PR TITLE
building the interface to update department members

### PR DESCRIPTION
**Pull request by ngoquocan203**

### CODE REVIEW

File: resources/js/Pages/Departen/User.vue [hunk 1]
```diff
@@ -27,15 +27,24 @@ const closeModal = () => {
...
+const user = ref('')
 onBeforeMount(async () => {
     await axios.get(`/user/list`).then((res) => {
         users = res.data.users
     })
 });
 
-
-
+const role = ref('')
```
Comment (phamkykhoi) [inline resources/js/Pages/Departen/User.vue:37]: role => roles

Comment (ngoquocan203) [reply resources/js/Pages/Departen/User.vue:37]: fixed

File: resources/js/Pages/Departen/User.vue [hunk 2]
```diff
@@ -27,14 +27,14 @@ const closeModal = () => {
 };
 
 let users = reactive([]);
-const user = ref('')
+const listUser = ref('')
```
Comment (phamkykhoi) [inline resources/js/Pages/Departen/User.vue:30]: listUser
=>
users

Comment (ngoquocan203) [reply resources/js/Pages/Departen/User.vue]: fixed

**Pull request merged**